doc/sphinx/proof-engine/ltac.rst

```diff
@@ -1151,34 +1639,51 @@ Tactic toplevel definitions
 Defining |Ltac| functions
```

Comment:
I'm not convinced the term "function" should be used since there may be no parameters. I would say "Defining toplevel |Ltac| objects" instead. (If this is changed, it should also be adapted in the description.)

Reply:
What do you have against functions with no parameters?  Most languages support them.  Most of our users won't be OCaml programmers.

Reply:
I don't have anything against them, but there is already a notion of function in `ltac_expr`, it is `fun ... => ...`. `Ltac foo parameters := bar` is a shorthand for `Ltac foo := fun parameters => bar` so it does define a function but `Ltac foo := bar` doesn't.

Reply:
OK.  BTW, see #12469